setup.cfg

```diff
@@ -13,18 +13,18 @@ classifiers =
     Intended Audience :: Developers
     License :: OSI Approved :: MIT License
     Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.6
-    Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
 
 [options]
-python_requires = >=3.6
+python_requires = >=3.8
```

Comment:
I'm wondering why this is needed. As things seem to work properly for 3.6.

Reply:
@J535D165 ok to merge?